File: JS/URI/1117.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');


let a;
let cont = 0;
let media = 0;
let j=0;

while (cont < 2) {
    a = parseFloat(lines[j]);
    if (a >= 0 && a <= 10) {
        media = media + a;
        cont++;
    } else console.log("nota invalida");
    j++;
}
media = media / 2;
console.log("media = "+ media.toFixed(2));

File: Python/URI/1117.py
a = 0
cont = 0
media = 0
j = 0

while (cont < 2):

    a = float(input())
    if (a >= 0.0 and a <= 10.0):

        media = media + a
        cont = cont + 1
    else:
        print("nota invalida")

media = media / 2
print("media = %0.2f" % media)
